index.html

```diff
@@ -4,164 +4,188 @@
+        <div class="container">
+          <div class="mb-5">
+            <h5 class="text-center h2 text-lightblue">Latest Bikes</h5>
           </div>
         </div>
         <div class="">
-          <div class="">
-            <div class="">
-              <div class="">
-
+          <div class="grid-container-div">
+            <div class="grid-item">
+              <div class="grid-img">
+                <img src="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/harley-fxdr-1540482462.png?crop=1xw:1xh;center,top&resize=5040:*" alt="" class="w-95">
               </div>
-              <div class="">
-
+              <div class="grid-content">
+                <h6 class="h5">Harley-Davidson FXDR 114</h6>
+                <p class="text-lightgray"><span class="h6">Base Price:</span> $21,349</p>
               </div>
             </div>
-            <div class="">
-              <div class="">
-
+            <div class="grid-item">
+              <div class="grid-img">
+                <img src="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/indiant-ftr-1200-1540482461.png?crop=1xw:1xh;center,top&resize=5040:*" alt="" class="w-95">
               </div>
-              <div class="">
-
+              <div class="grid-content">
+                <h6 class="h5">Indian FTR 1200</h6>
+                <p class="text-lightgray"><span class="h6">Base Price:</span> $12,999</p>
               </div>
             </div>
-            <div class="">
-              <div class="">
-
+            <div class="grid-item">
+              <div class="grid-img">
+                <img src="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/bmw-1540482626.png?crop=1xw:1xh;center,top&resize=5040:*" alt="" class="w-95">
               </div>
-              <div class="">
-
+              <div class="grid-content">
+                <h6 class="h5">BMW G310GS</h6>
+                <p class="text-lightgray"><span class="h6">Base Price:</span> $5,695</p>
               </div>
             </div>
-            <div class="">
-              <div class="">
-
+            <div class="grid-item">
+              <div class="grid-img">
+                <img src="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/yamaha-1540482460.jpg?crop=1xw:1xh;center,top&resize=5040:*" alt="" class="w-95">
               </div>
-              <div class="">
-
+              <div class="grid-content">
+                <h6 class="h5">Yamaha YZF-R3</h6>
+                <p class="text-lightgray"><span class="h6">Base Price:</span> $4,999</p>
               </div>
             </div>
           </div>
-          <div class="">
-            <div class="">
-              <div class="">
-
+        </div>
+      </div>
+      <div class="container">
+        <div class="container">
+          <div class="mb-5">
+            <h5 class="h2 text-center text-lightblue">Parts on Shop</h5>
+          </div>
+        </div>
+        <div class="">
```

Comment:
do not leave empty class, fill it or remove it.

Reply:
we removed them.